test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -24,6 +24,7 @@ namespace Microsoft.ML.Runtime.RunTests
     /// </summary>
     public abstract partial class BaseTestBaseline : BaseTestClass, IDisposable
     {
+        public const decimal tolerance = 10000000;
```

Comment:
When we first introduced it , most of the numbers on unix differed on 7th or more decimal place. so Zeeshan S suggested to use this

Reply:
Ok. Sounds good. 

I walked down the code, and we're using the `precision` here:
https://github.com/dotnet/machinelearning/blob/a02807c7a805b72ef12970a37279c8cec4ea667d/test/Microsoft.ML.TestFramework/BaseTestBaseline.cs#L630-L639

Odd that the formula isn't symmetric. And if `f2 == 0`, no room is given to deviation/rounding errors; I suppose we haven't run into this case.

Reply:
yeah you are right, it will be a problem if ```f2==0```
But In most cases , f2 == 0 will be when f1 == 0 because we are generally multiplying and adding random numbers and probablity of all of them leading to exact zero will be quite small.

However we can change this function in future, if we face such issues. The function is quite independent from rest of everything. we can plug any function down the line if this one is not working upto our standards